### DISCUSSION TITLE
Way to change avatar?

### DISCUSSION THREAD

**Discussion by csm10495** [ValvePython/steam#327] [Q&A]
I see client.user.get_avatar_url(), though is there a way to set your avatar with this module?

**Comment by rossengeorgiev** [comment]
No

**Comment by XronoZ-create** [comment]
Can i make PR for that?

**Comment by rossengeorgiev** [comment]
@XronoZ-create feel free to make a PR

**Comment by xiao-e-yun** [comment]
http://steamcommunity.com/profiles/<usersteam-id\>/minimaledit 
or
https://steamcommunity.com/id/<custom-steam-link\>/minimaledit

maybe you can get something

and
https://steamcommunity.com/miniprofile/<account-id\>/json
you can get 
    persona_name
    avatar_url
